AndroidManifest.xml

```diff
@@ -408,6 +408,11 @@
         android:authorities="${applicationId}.attachments"
         android:grantUriPermissions="true"
         android:exported="false">
+
+      <meta-data
```

Comment:
please add some comment - here or  in the PR pr commit description.

some summary from the chat yesterday, why is that needed, what happens without. but also: who consumes this ALLOW_INTERNAL_ACCESS? android? contentresolver? safecontentresolver?

i already forgot today what exactly happens :) -- let's be kind to the maintainers next years stumbling upon that code, maybe it's our future-self :)

Reply:
> some summary from the chat yesterday, why is that needed, what happens without. but also: who consumes this ALLOW_INTERNAL_ACCESS? android? contentresolver? safecontentresolver?

I don't know, my guess is that this is a hint for the safecontentresolver lib to recognize which providers are whitelisted and allow them